functions/go/fix/README.md

```diff
@@ -0,0 +1,95 @@
+# fix
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Fix resources and make them compatible with kpt 1.0.
+
+<!--mdtogo-->
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+`fix` helps you migrate the resources from `v1alpha1` format to `v1alpha2` format.
```

Comment:
With something like this, documenting the limitations (What it cannot do) is as important as what it can do

Reply:
Updated the limitations.